### ISSUE TITLE
Paper gauss-newton

### ISSUE THREAD

**Issue by gftabor** [symforce-org/symforce#206]
Unclear if this is really correct place to say this, but the Gauss-Newton update step is missing the inverse of the hessian in the paper.
![image](https://user-images.githubusercontent.com/14334346/182422349-e4b0f29c-3ab9-42a8-b8ce-619a28af3dab.png)
https://en.wikipedia.org/wiki/Gauss%E2%80%93Newton_algorithm

Not a direct comparison but in the Levenberg marquardt you use the cholesky inverse here https://github.com/symforce-org/symforce/blob/a1da5b07e287a3130c1fac4e5f1bb14a787ad1b2/symforce/opt/levenberg_marquardt_solver.tcc#L194

tldr; Code is right, Paper is wrong?

**Comment by aaron-skydio** [comment]
Thanks for reporting!  Yeah we definitely compute the inverse (factorize + solve :slightly_smiling_face: ) in the code, just a typo in the paper
